File: src/lugbygame/Order.java
package lugbygame;

//주문
public class Order {
	// dx,dy of 각 player(0-5)
	// passto
	// initialize
	int[] dx = new int[6];
	int[] dy = new int[6];
	int passto = -1;

	public void clear() {
		dx = new int[6];
		dy = new int[6];
		int passto = -1;
	}

	public Order() {
	}

	public Order(Order o, boolean mirror) {
		dx = o.dx;
		dy = o.dy;
		passto = o.passto;

		for (int i = 0; i < dx.length; i++) {
			dx[i] *= -1;
		}
	}
}

File: lucby_edit/Order.py
class Order:
	dx=[0 for _ in range(6)]
	dy=[0 for _ in range(6)]
	passto =-1

	def clear(self):
		self.dx = [0 for _ in range(6)]
		self.dy = [0 for _ in range(6)]
		self.passto = -1

	
	def __init__(self,o=None, mirror=None):
		if o != None:
			self.dx = o.dx
			self.dy = o.dy
			self.passto = o.passto
			for i in range(len(self.dx)):
				self.dx[i] *= -1
		else:
			pass


